docs/apm/real-user-monitoring.md

```diff
@@ -366,8 +366,9 @@ The content then populates in **Sumo Logic RUM - default** dashboards (inside
 Once you've installed the RUM app, use our [Explore view](/docs/dashboards-new/explore-view.md) to gain visibility into your web app's performance and end-user activity such as geographic location, browser type, operating systems used. These dashboards visualize RUM metrics gathered from browser tracing instrumentation.
 
 Explore organizes RUM data on four levels:
-* **Application**: corresponds to the value of the application tag set in your RUM JavaScript script above. This should correspond to your whole website defined by its business function, such as "Coffee shop".
-* **Service**: corresponds to the name of the service in your RUM JavaScript script above. This should correspond to a JS code executed in the browser, such as "coffee-shop-web". You can have multiple services for each application. 
+* **Application**: corresponds to the value of the application tag set in your [RUM JavaScript script](#step-2-add-script-to-your-page-header). This should correspond to your whole website defined by its business function, such as `Coffee shop`.
+* **Service**: corresponds to the name of the service in your [RUM JavaScript script](#step-2-add-script-to-your-page-header). The value should correspond to the JavaScript code executed in your browser, such as `coffee-shop-web`. You can have multiple services for each application. 
+* **Environment**: corresponds to your development environment, like `prod` or `dev`. To enable this, add the `deployment.environment` tag and desired value to your [RUM JavaScript script](#step-2-add-script-to-your-page-header).
```

Comment:
@pbrzoska-sumo , should we also add Application, Service, and Environment variables to the [JavaScript example snippets](https://help.sumologic.com/docs/apm/real-user-monitoring/#step-2-add-script-to-your-page-header) earlier in the doc?

Reply:
Service is already there as the only mandatory parameter. We can add application and environment to make it more complete, but we should do that in all places, including https://github.com/SumoLogic/sumologic-opentelemetry-js and all methods (Synchronous, Asynchronous, NPM). If you have the time for that - go ahead and please work with dev team.

Reply:
I'll create a separate ticket for this